### PR TITLE
added hard coded "capacity" metadata capacity field

### PR DESCRIPTION
**Pull request by danhanson**
This adds a capacity to the routing request

### CODE REVIEW

**Comment by danhanson** [comment]
Ooops I forgot the outer array for capacities

File: app/io/pathfinder/routing/ClusterRouter.scala [hunk 1]
```diff
@@ -201,10 +201,12 @@ class ClusterRouter(clusterId: Long) extends EventBusActor with ActorEventBus wi
 
             val comTable = JsObject(commodities.indices.map(num => ((num+1).toString,JsNumber(num+commodities.size+1))))
             val vehicleTable = JsArray(vehicles.indices.map(num => JsNumber(num+2*commodities.size+1)))
+            val capacities = JsArray(commodities.map(_.metadata.validate((__ \ "capacity").read[JsNumber]).getOrElse(JsNumber(0)))
+                                  ++ vehicles.map(_.metadata.validate((__ \ "capacity").read[JsNumber]).getOrElse(JsNumber(0))))
```
Comment (aj-michael) [inline app/io/pathfinder/routing/ClusterRouter.scala]: I think vehicles without capacities should default to Integer.MAX_VALUE, not 0.

File: app/io/pathfinder/routing/ClusterRouter.scala [hunk 2]
```diff
@@ -11,7 +11,7 @@ import io.pathfinder.websockets.pushing.EventBusActor
 import io.pathfinder.websockets.{Events, ModelTypes}
 import play.Logger
 import play.api.Play
-import play.api.libs.json.{JsResultException, JsString, Json, JsArray, Reads, __, JsNumber, JsObject, Writes, JsValue}
+import play.api.libs.json.{JsResultException, JsString, JsArray, Reads, __, JsNumber, JsObject, Writes, JsValue}
```
Comment (aj-michael) [inline app/io/pathfinder/routing/ClusterRouter.scala]: Is __ really a thing you can import? lol

Comment (danhanson) [inline app/io/pathfinder/routing/ClusterRouter.scala]: Yeah, it is for json parsing, you can also use variables named $ or %, and even < too if you want.

**Comment by aj-michael** [comment]
LGTM :+1:

**Pull request merged**